runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -76,7 +85,39 @@ public void onLookupMenuClicked(PlayerMenuOptionClicked event)
 	{
 		if (event.getMenuOption().equals(LOOKUP))
 		{
-			executor.execute(() ->
+			lookup(event.getMenuTarget());
+		}
+	}
+	
+	@Subscribe
+	public void onMenuEntryAdded(MenuEntryAdded event)
+	{
+		if ((event.getOption().equals("Add friend") || event.getOption().equals("Message")) && !event.getTarget().equals(""))
```

Comment:
I think a more generic way to add menu options, as part of MenuManager, would be best.
```
menuManager.addAfter("Attack", LOOKUP);
```
Or so. But I'd be okay merging this first regardless.

Reply:
Yes, that would be a better solution. Unfortunately, I don't know how to do that. Could need some help to get started.